pyanaconda/modules/network/firewall/firewall.py

```diff
@@ -203,3 +204,18 @@ def set_disabled_services(self, disabled_services):
         self._disabled_services = list(disabled_services)
         self.disabled_services_changed.emit()
         log.debug("Services that will be explicitly disabled on the firewall: %s", self._disabled_services)
+
+    def install_with_task(self, sysroot):
```

Comment:
Please, use the predefined method `install_with_tasks`.

Reply:
I didn't realize that this is a submodule of the Network module. It can stay like this.